lib/Rx/Observable/ErrorObservable.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Rx\Observable;
+
+use Exception;
+use Rx\Disposable\EmptyDisposable;
+
+class ErrorObservable extends BaseObservable
```

Comment:
Yeah, I'll get rid of the other one. I like `Error` better. We can follow up with that later though. See @mbonneau's comment below.